models/lit.cfg.py

```diff
@@ -0,0 +1,31 @@
+import os
+import lit.formats
+
+config.name = "Halo Model Tests"
+
+# Setup source root.
+config.test_source_root = os.path.dirname(__file__)
+
+# Test suffixes.
+config.suffixes = ['.sh']
+
+config.excludes = ['run.all.sh', 'get_images.sh']
+
+config.environment['SRC_DIR'] = config.halo_src_dir
+config.environment['BUILD_DIR'] = config.halo_build_dir
+config.environment['ODLA_INC'] = os.path.sep.join(
+    (config.halo_src_dir, 'ODLA', 'include'))
+config.environment['ODLA_LIB'] = os.path.sep.join(
+    (config.halo_build_dir, 'lib'))
+config.environment['TEST_TEMP_DIR'] = '/tmp'
+config.environment['TEST_WITH_GPU'] = '1'
```

Comment:
check if based on the availability of libodla_tensorrt?

Reply:
yes, I have used cmake option ""@cudart@"" to check it